rmgpy/rmg/main.py

```diff
@@ -716,10 +725,16 @@ def execute(self, **kwargs):
                         else:
                             self.updateReactionThresholdAndReactFlags()
                         
+                        self.reactionModel.setThermodynamicFilteringParameters(self.Tmax, toleranceThermoKeepSpeciesInEdge=modelSettings.toleranceThermoKeepSpeciesInEdge,
```

Comment:
Is it possible to have a check that prevents calling `setThermodynamicFilteringParameters` and `thermoFilterDown` if the user does not specify thermodynamic pruning. This could prevent any errors encountered in thermo filtering if the user does not desire to use that functionality.